README.md

```diff
@@ -3,51 +3,54 @@
+Une fois que vous avez repéré une ISSUE, assignez vous l'ISSUE, et vous commencerez par créer votre branche portant le nom de ISSUE.
 
 Exemple : ``git checkout -b ISSUE-#22``
 
-Vous effectuerai votre travail dessus après avoir commit (en suivant toujours la convention de nommage ```ISSUE-#22 : <la tache réaliser au sens précis>```) et push votre travail, vous pouvez passez a l'etape suivante
-qui est d'ouvrir une Pull Request on imposera une convention en la nommant 
+Vous effectuerez votre travail dessus. Après avoir commit (en suivant toujours la convention de nommage ```ISSUE-#22 : <la tache réaliser au sens précis>```) 
+et push votre travail, vous pouvez passez a l'etape suivante,
+qui est d'ouvrir une Pull Request. On imposera une convention en la nommant :
 
 ```ISSUE-#22 : <la tache réaliser au sens large >```
 
-Concentrez vous vraiment sur la tâche choisi ne mélangez pas une correction que vous auriez envie de faire dans une tâche qui n'a rien n'a voir.
-Il est souvent facile de s'éparpiller..
-Si vous estimez que vous avez fini votre travail vous pouvez le labelisser en "Pret pour relecture"
-un/des contributeurs pourront relire et proposer des corrections/suggestions à apporter à votre travail.
-Une fois celui-ci validé par au moins 2 contributeurs votre travail sera en mesure d'être merger dans le master.
-Vous pourrais ensuite retourner sur votre master en local et effectuer un ```git pull```.
+Concentrez vous vraiment sur la tâche choisie. Ne mélangez pas une correction que vous auriez envie de faire dans une tâche qui n'a rien à y voir.
+Il est souvent facile de s'éparpiller...
+Si vous estimez que vous avez fini votre travail, vous pouvez le labelliser en "Prêt pour relecture".
+Un/Des contributeurs pourront relire et proposer des corrections/suggestions à apporter à votre travail.
+Une fois celui-ci validé par au moins 2 contributeurs, votre travail sera en mesure d'être mergé dans le master.
+Vous pourrez ensuite retourner sur votre master en local et effectuer un ```git pull```.
 
-Pensez à supprimer vos "branch" pour ne pas finir avec d'innombrable branch incompréhensible.
+Pensez à supprimer vos "branch" pour ne pas finir avec un nombre incompréhensible de branches.
 
 ## INSTALLATION
 
 ### Paypal SANDBOX TEST
 
-Crée votre SANDBOX (celle-ci vous permettra d'effectuer des faux paiements et avoir le réel comportement de l'application) :
+Créez votre SANDBOX (celle-ci vous permettra d'effectuer des faux paiements et avoir le réel comportement de l'application) :
 
 Cf : https://developer.paypal.com/docs/api/overview/#create-sandbox-accounts
 
-Crée ensuite un fichier .env.local avec comme variable d'environnement:
+Créez ensuite un fichier ``.env.local`` avec comme variable d'environnement:
 
 ```
 CLIENT_ID=
 CLIENT_SECRET=
 ```
 
-Après avoir configuré le .env.local du projet :
+Après avoir configuré le ``.env.local`` du projet :
 
 ```
 $ git clone git@github.com:senzowayne/resavo.git
```

Comment:
git clone git@github.com:<username>/resavo.git

Reply:
Peut-être préciser, avant, qu'il faut forker le projet ? :thinking:

Reply:
Dans la doc il le font pour Symfony mais on est pas obligé de fork resavo enfin je veux dire c'est libre à chacun

Reply:
Mais pour avoir `git@github.com:USERNAME/resavo.git`, il faut forker le repo, non ? :thinking:

Reply:
https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository

Reply:
J'ai testé `git@github.com:Dev-Int/resavo.git` sans le forker => `ERROR: Repository not found.`

Reply:
Ah d’accord bon ben rajoute le fork pas de souci façon ça coûte rien 😉

Reply:
Le fork n'est pas nécessaire pour pouvoir utiliser le repository :)

Reply:
C'était justement la question @Pierstoval 🤣
Je ne suis pas au top sur l'utilisation de GitHub... 🤔
Ouvert à la discussion... :wink: